tools/create-app.bash

```diff
@@ -3,25 +3,29 @@
 # Create a script that given an output directory builds the current application
 # and generates an .aab file in the output dir.
 
-# Params: $1: output directory
+### Parameters ###
+
+OUTPUT_DIRECTORY = $1
+ASSETS = $2
```

Comment:
Can we rename it to ASSETS_DIRECTORY so that it matchs the other variable names?